### PR TITLE
Move non-gofer files to katello-host-tools

### PR DESCRIPTION
**Pull request by jlsherrill**

### CODE REVIEW

File: katello-agent/katello-agent.spec
Comment (sean797) [inline katello-agent/katello-agent.spec]: s/prrata/errata/

**Review by sean797 — CHANGES_REQUESTED** [review]

File: katello-agent/katello-agent.spec
Comment (ehelms) [inline katello-agent/katello-agent.spec]: Nitpick: Capital Katello

File: katello-agent/katello-agent.spec
Comment (ehelms) [inline katello-agent/katello-agent.spec]: Will this result in `katello-agent-fact-plugin` or the subpackage name being used? `katello-host-tools-fact-plugin` ?

File: katello-agent/katello-agent.spec
Comment (ehelms) [inline katello-agent/katello-agent.spec]: Mind aligning with the other two fields?

File: katello-agent/katello-agent.spec
Comment (ehelms) [inline katello-agent/katello-agent.spec]: Does this upper level package actually require subscription-manager or just the fact-plugin?

File: katello-agent/katello-agent.spec
Comment (jlsherrill) [reply katello-agent/katello-agent.spec]: yes, let me obsolete the old one

File: katello-agent/katello-agent.spec
Comment (jlsherrill) [reply katello-agent/katello-agent.spec]: yes, all 3 do as they all call into subman in different ways.

**Comment by jlsherrill** [comment]
Updated to move katello-host-tools as the base package, and katello-agent as the sub-package

**Comment by sean797** [comment]
Thanks @jlsherrill! 2 questions...
If I am reading this right if a users wants goferd clients you would do the usual `yum install katello-agent` and if they want gofer-less clients you would do `yum install katello-host-tools` ?

Do you think its worth renaming the katello-agent repo to katello-host-tools ?

**Comment by jlsherrill** [comment]
@sean797 exactly!  I do think its worth renaming the katello-agent git repo.  I was just waiting till after this went in to do it.   I'll send an email to foreman-dev to alert everyone of the pending change and proceed after a couple days.

**Comment by ehelms** [comment]
We should ensure that a package upload and an enabled repository upload occur when the `katello-host-tools` package is installed.

**Comment by ehelms** [comment]
I'd like to merge this once https://github.com/theforeman/forklift/pull/478 has been approved to ensure we have testing in place.

File: katello-host-tools/katello-host-tools.spec
Comment (sean797) [inline katello-host-tools/katello-host-tools.spec:7]: Do you want to update this now or in another PR after?

**Review by sean797 — APPROVED** [review]
ACK from me, thanks @jlsherrill, nice feature 👍

File: katello-host-tools/katello-host-tools.spec
Comment (jlsherrill) [reply katello-host-tools/katello-host-tools.spec:7]: If this gets merged in the next ~24 hours, i'll do a followup pr.  Otherwise I'll update before merging.

**Review by ehelms — APPROVED** [review]

**Comment by ehelms** [comment]
@jlsherrill ACK'd -- when you want to merge and do the necessary koji work feel free. Just ping me or also merge the Forklift PR.

**Comment by jlsherrill** [comment]
@ehelms updated with:

```
diff --git a/katello-host-tools/katello-host-tools.spec b/katello-host-tools/katello-host-tools.spec
index f40b743..78b668c 100644
--- a/katello-host-tools/katello-host-tools.spec
+++ b/katello-host-tools/katello-host-tools.spec
@@ -133,11 +133,11 @@ touch /tmp/katello-agent-restart
 exit 0
 
 %posttrans
-katello-package-upload &> /dev/null
-
+katello-package-upload 2> /dev/null
+katello-enabled-repos-upload 2> /dev/null
 exit 0
 
-%postun
+%postun -n katello-agent
 touch /tmp/katello-agent-restart
 exit 0
 
```

**Comment by ehelms** [comment]
ACK

**Comment by jlsherrill** [comment]
@ehelms all built in koji and ready to go

**Comment by ehelms** [comment]
@jlsherrill getting repoclosure errors: http://ci.theforeman.org/view/Katello%20Pipeline/job/release_repoclosure_katello_client_nightly_el7/355/console

**Pull request merged**
